### ISSUE TITLE
Questions about the transformation of various points in the frame

### ISSUE THREAD

**Issue by LL884800** [PRBonn/MapClosures#40]
Hello, I would like to ask you something. In this algorithm, the pose of the frame is estimated through Kiss-ICP. Is this estimated pose directly assumed to be the pose corresponding to the first time in the frame? Or is it basically not considering the inconsistency of the time series of each point in the frame, and directly converting the bits of all points in the frame to the frame

**Comment by saurabh1002** [comment]
Hey @LL884800 ,

Thanks for your interest in our work.

However, I could not understand your question, especially the last line. What do you mean by "bits of all points in the frame"? Could you rephrase your question in an elaborate way?

As a side note, we explain in the paper about the reference frame used for the local maps.

**Comment by LL884800** [comment]
Hello, what I would like to ask is that for this algorithm, the pose estimation between point cloud files is obtained using kiss-icp. How do you understand the pose here? It is the first point in the point cloud file. Is it the corresponding pose? Does this algorithm involve time series interpolation to process each point in each point cloud file? How to understand the coordinate transformation of each point in the point cloud file?

**Comment by saurabh1002** [comment]
There is no deskewing operation being performed on the pointclouds. So, there is no pose associated with every point in the pointcloud.

Assume that scans with index `i` to `n` belong to a local map, and we have corresponding odometry poses from KISS-ICP as `T_i' to `T_n`, then the local map is in the **local reference frame** of scan `i` associated with pose `T_i`. The map is just a voxel down sampled pointcloud of all the scans `i` to `n` in the local reference frame.

**Comment by saurabh1002** [comment]
I would refer you to the paper for more such details, as well as the discussion we had earlier, where I try to explain the same stuff.
